packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -205,7 +220,7 @@ export class LitContracts {
     this.network = args?.network || 'cayenne';
     // if rpc is not specified, use the default rpc
     if (!this.rpc) {
-      this.rpc = DEFAULT_RPC;
+      this.rpc = LIT_RPC.VESUVIUS;
```

Comment:
This feels like a breaking change for everyone who is using the contracts SDK against our existing habanero, cayenne and manzano networks.  Can we really change this without breaking existing SDK users using those networks?  I think we should set the default based on `network` instead of changing the default value -- WDYT?

Reply:
yea that makes sense!